fix-point/src/Rename.hs

```diff
@@ -19,24 +19,27 @@ TIO.putStrLn $ rename "CHANGE" "import Lib.App (AppEnv (..))\n\
 :}
 -}
 changeLine :: Text -> Text -> Text
-changeLine newMod line = case words line of
-    -- module names
-    "module" : s : extra     -> unwords ("module": prefix s : extra)
-
-    -- imports which start with Lib
-    "import" : s : extra     -> if take libLen (toString s) == "Lib"
-                                then unwords ("import": prefix s : extra)
-                                else line
-
-    -- reexport where reexport module started with Lib
-    "(":"module" : s : extra -> unwords ("(" :"module": prefix s : extra)
-    ",":"module" : s : extra -> unwords ("," :"module": prefix s : extra)
-
-    -- Anything else can't be changed
-    _ -> line
+changeLine newMod line = decide line
```

Comment:
What is the motivation behind this refactoring?
Also, you can eta-reduce and write just `changeLine newMod = decide`